cypress/support/commands/navigation.js

```diff
@@ -51,16 +51,16 @@ Cypress.Commands.add('selectSingleTabRow', () => {
 Cypress.Commands.add('openReferencedDocuments', (referenceId, retriesLeft = 8) => {
   // retry 8 times to open the referenced document
   const date = humanReadableNow();
+  const timeout = { timeout: 20000 };
 
   if (retriesLeft >= 1) {
     const referencesAliasName = `references-${date}`;
     cy.server();
     cy.route('GET', new RegExp(RewriteURL.REFERENCES)).as(referencesAliasName);
 
     cy.get('body').type('{alt}6'); // open referenced docs
-    cy.wait(`@${referencesAliasName}`, { timeout: 20000 });
-
-    cy.get('.order-list-panel .order-list-loader').should('not.exist');
+    cy.wait(`@${referencesAliasName}`, timeout);
+    cy.get('.order-list-panel .order-list-loader', timeout).should('not.exist');
 
```

Comment:
I have also tried this * cy.get('.order-list-panel .order-list-loader', timeout).should('not.exist'); * previously when we had the errors regarding openReferencedDocuments() but it didn't work and I have seen that sometimes on jenkins, there is one test which is still failing with that  **CypressError: Timed out retrying: Expected <div.order-list-loader.text-center> not to exist in the DOM, but it was continuously found.**  (https://dashboard.cypress.io/#/projects/5yp4q1/runs/489/specs - https://jenkins.metasfresh.com/job/ops/job/run_e2e_tests/599/console). let's keep an eye on this in the future. Somehow I have the feeling that the problem is not entirely solved but maybe this is not the real issue.

Reply:
One way or another we're gonna make this shit work properly. **by force if there's no other way** 💪